src/Components/Search/Suggestions/SuggestionItem.tsx

```diff
@@ -12,29 +12,57 @@ interface Props {
   query: string
 }
 
-export const SuggestionItem: SFC<Props> = props => {
-  const { label, href, isHighlighted } = props
+interface FirstProps {
+  totalCount: number
+  display: string
+  href: string
+  isHighlighted: boolean
+  label: string
+  query: string
+}
+
+export const FirstSuggestionItem: SFC<FirstProps> = props => {
+  const { href, totalCount, isHighlighted, query } = props
+  const boxStyle = {
+    borderBottom: `1px solid ${color("black10")}`,
+  }
+
+  return (
+    <Box bg={isHighlighted ? "black5" : "white100"} style={boxStyle}>
+      <Link color="black100" href={href} underlineBehavior="none">
+        <SuggestionWrapper>
+          <InnerWrapper
+            flexDirection="column"
+            flexGrow="1"
+            justifyContent="center"
+          >
+            See full results for "{query}"
+          </InnerWrapper>
+          <Flex flexGrow="0" px={2}>
+            <Serif size="2" color="black30">
+              {totalCount} results
```

Comment:
Try `totalCount.toLocaleString()` for the comma.

Reply:
TY!! ❤️